### PR TITLE
Managed Connector docs

### LINKED ISSUE #1209: Add Connector content based on available runbooks
We've had numerous requests for additional connector docs. There are already numerous runbooks available. Take these runbooks and add to our existing connector doc (of course, format each one identically). Biggest ask here is to make sure engineering has tested all. 

See https://github.com/redpanda-data/runbooks/tree/main/content/howtos/connectors/managed for the content to be leveraged.

See https://docs.google.com/spreadsheets/d/1bvu4j5RDEkIUz2rOse0M3Ewxmm1WryQcz2oUE-Jt3_A/edit#gid=0 for the prioritization. Connector doc should be put out in order of this list. Ideally, the PR includes all.

cc: @bpraseed

### PR DESCRIPTION
**Pull request by Feediver1**
Resolves #1209

Connector docs based on the runbooks available at https://github.com/redpanda-data/runbooks/tree/main/content/howtos/connectors/managed.

Preview: https://deploy-preview-1259--redpanda-documentation.netlify.app/docs/beta/deploy/deployment-option/cloud/managed-connectors/

### CODE REVIEW

File: docs/deploy/deployment-option/cloud/managed-connectors/index.mdx [hunk 1]
```diff
@@ -9,4 +9,33 @@ title: Managed Connectors
 
 - [Amazon S3 Sink Connector](../managed-connectors/create-s3-sink-connector)
 
-    A connector subscribes to specified Kafka topics and collects messages. The connector periodically dumps the messages to a specified location, such as an S3 bucket or a GCS bucket. You can create a connector, managed by Redpanda, directly from the Redpanda Console.
\ No newline at end of file
+    A connector subscribes to specified Kafka topics and collects messages. The
+    connector periodically dumps the messages to a specified location, such as an
+    S3 bucket or a GCS bucket. You can create a connector, managed by Redpanda,
```
Comment (tomasz-sadura) [inline docs/deploy/deployment-option/cloud/managed-connectors/index.mdx:14]: S3 sink connector push data to S3 only (not to GCS).

File: docs/deploy/deployment-option/cloud/managed-connectors/create-gcs-sink-connector.mdx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+  * A cluster provisioned in Redpanda Cloud.
+  * Login credentials to GCS.
+
+* You must enable the following object permissions in the bucket:
+
+  * `storage.objects.create`
+  * `storage.objects.delete` (needed for overwriting)
+
+**^^ These properties are not described in our docs, and are not included in the properties tables below. We should provide the descriptions somewhwere.**
```
Comment (tomasz-sadura) [inline docs/deploy/deployment-option/cloud/managed-connectors/create-gcs-sink-connector.mdx:25]: These are the Google Cloud permissions, not the connector properties. They must be set in Google Cloud console.

File: docs/deploy/deployment-option/cloud/managed-connectors/create-bigquery-sink-connector.mdx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+   | `keySource` | Determines whether the `keyfile` config is the path to the credentials JSON file (`FILE`) or the raw JSON of the key itself (`JSON`).  |
+   | `keyfile` | The file containing a JSON key with BigQuery service account credentials, or the credentials in JSON, depending on the `keyfile` value. |
+   | `project` | The BigQuery project to write to. |
+   | `defaultDataset` | The default dataset to be used. |
+   | `autoCreateTables` | Automatically create BigQuery tables if they don't already exist. |
+
+### Map the topic name to the table name
+
+By default the table name is the name of the topic. You may need to use different
+table names (**When or why would users need to do this?**):
```
Comment (bochenekmartin) [inline docs/deploy/deployment-option/cloud/managed-connectors/create-bigquery-sink-connector.mdx:61]: It's when the user needs different topic name than a table name. For example given topic name already used for a different purpose.

File: docs/deploy/deployment-option/cloud/managed-connectors/create-bigquery-sink-connector.mdx [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+  ```js
+  {
+    "transforms": "dropPrefix",
+    "transforms.dropPrefix.type": "org.apache.kafka.connect.transforms.RegexRouter",
+    "transforms.dropPrefix.regex": "dbserver\\.public\\.(.*)",
+    "transforms.dropPrefix.replacement": "$1"
+  }
+ ```
+
+ **Are there instructions for testing the connection? Troubleshooting?**
```
Comment (bochenekmartin) [inline docs/deploy/deployment-option/cloud/managed-connectors/create-bigquery-sink-connector.mdx:83]: BigQuery connector Troubleshooting added recently in:
https://runbooks.cloud.redpanda.com/howtos/connectors/managed/bigquery-sink-connector/#troubleshooting

**Comment by Feediver1** [comment]
**STATUS:** The "advanced" JSON configs for managed connectors will be available when Redpanda releases an API for connector configs some time in Q1/Q2 2023. At that time, this material can be used/leveraged for the API doc--in other words, an advanced configuration tool that for now we do not want to expose to users. Instead, I will circle back to the updated Connector UI and create docs based on the wizard for each connector, leveraging the updated runtime docs, which are more consistent now. Closing until we have the API.

**Pull request closed without merging**